armory/utils/export.py

```diff
@@ -1,74 +1,51 @@
 import os
 import logging
+import abc
 import numpy as np
 import ffmpeg
 import pickle
 import time
 from PIL import Image, ImageDraw
 from scipy.io import wavfile
 
-from armory.data.datasets import ImageContext, VideoContext, AudioContext, So2SatContext
-
 
 logger = logging.getLogger(__name__)
 
 
 class SampleExporter:
```

Comment:
Just a thought: do we want the ability to export just the annotated ground truth?